context.go

```diff
@@ -445,6 +445,24 @@ func (ctx *Context) JSON(jsonObjectOrArray interface{}) error {
 	return ctx.WriteJSON(http.StatusOK, jsonObjectOrArray)
 }
 
+// ReadXML reads XML from request's body
+func (ctx Context) ReadXML(xmlObject interface{}) error {
```

Comment:
Should we tests the errors ? Line 452, 460 probably they will be missing in coverage.